Mathlib/Analysis/Normed/Module/Completion.lean

```diff
@@ -27,7 +27,7 @@ namespace UniformSpace
 
 namespace Completion
 
-variable (𝕜 E : Type*) [NormedField 𝕜] [NormedAddCommGroup E] [NormedSpace 𝕜 E]
+variable (𝕜 E : Type*) [NormedField 𝕜] [SeminormedAddCommGroup E] [NormedSpace 𝕜 E]
```

Comment:
I've done a much more thorough version of this in #17059, but they can be merged in either order.